mlir/include/mlir/Dialect/Tosa/IR/TosaOps.td

```diff
@@ -2071,17 +2075,17 @@ def Tosa_IfOp : Tosa_Op<"cond_if",
   }];
 
   let arguments = (ins
-    Tosa_I1Tensor:$cond,
+    Tosa_I1Tensor:$condition,
```

Comment:
Can we rename in a separate patch? Same for all other changes that do not relate with removing the Quantization information